java/src/jmri/InstanceManager.java

```diff
@@ -867,8 +867,12 @@ public static InstanceManager getInstanceManager() {
         /**
          * Replace the (static) InstanceManager.
          */
-        public static void resetInstanceManager() {
-            instanceManager = new InstanceManager();
+        public synchronized static void resetInstanceManager() {
```

Comment:
I'm concerned that changes to the core InstanceManager are being made in a "Spotbugs signal masts" PR.  Particularly when dealing with high-impact things, I'd prefer to have them separated out.  I guess this is done now, but please try to do this kind of thing on separate branches.

Reply:
Sorry, wasn’t intending to sneak this in as part of an open PR but must’ve had a different branch checked out than I thought I had. Will take this class out onto a separate branch/PR.

Reply:
Moved to PR #7068